doc/development/source/orange-demo/orangedemo/OWDataSamplerA.py

```diff
@@ -25,28 +25,7 @@ def __init__(self):
 
         # GUI
         box = gui.widgetBox(self.controlArea, "Info")
-        self.infoa = gui.widgetLabel(box, 'No data on input yet, waiting to get something.')
-        self.infob = gui.widgetLabel(box, '')
-# [end-snippet-1]
-
-# [start-snippet-2]
-    @Inputs.data
```

Comment:
Why is this part removed?

Reply:
I think the indentation was just changed, there shouldn't be any removals.

Reply:
That is strange. I will double check why that might have happened.

Reply:
OK, I have restored the lines. Sorry about the misstep.